src/store/reducers/storage/storage.ts

```diff
@@ -91,6 +94,7 @@ const storage: Reducer<StorageState, StorageAction> = (state = initialState, act
             return {
                 ...state,
                 visible: action.data,
+                usageFilter: [],
```

Comment:
Reset usage filter on selector change. This fixes the bug when you select some values in storage filter and then change selector to some value (e.g. "Out of space"), and receive empty table with no opportunity to update the selector